packages/server/modules/core/graph/resolvers/versions.ts

```diff
@@ -134,18 +134,23 @@ export = {
         project
       })
 
-      let lastVersion: Version | null
+      let latestVersion: Version | null = null
+      let latestVersions: Array<Version> | null = null
       if (getTypeFromPath(info) === 'Model') {
-        lastVersion = await ctx.loaders
+        latestVersion = await ctx.loaders
```

Comment:
this branching based on whether we're querying for the version through Model or not is no longer necessary